### ISSUE TITLE
UI CanvasScalar broken in game/play mode

### ISSUE THREAD

**Issue by Tryibion** [FlaxEngine/FlaxEngine#2338]
**Issue description:**
After this commit <https://github.com/FlaxEngine/FlaxEngine/commit/3b502086f897d99cf1ade8fa66ec7b47ffdd9b12> it looks like the new UI editor works, but when playing the game in the editor or cooked game, the controls show in the correct spot, but they are not triggering actions in the correct spots.

**Steps to reproduce:**
1. Create UI with Canvas Scalar with buttons.
2. Play in editor.
3. Try to select a button.

**Flax version:**
<https://github.com/FlaxEngine/FlaxEngine/commit/3b502086f897d99cf1ade8fa66ec7b47ffdd9b12>

**Comment by mafiesto4** [comment]
Yeah I made mistake. Should be fixed in 2b824ce22a4e120f745cae6b7f85589c4c0e92d9.
